src/api/posts/src/routes/feeds.js

```diff
@@ -34,6 +34,32 @@ feeds.get('/', async (req, res, next) => {
   );
 });
 
+//This should return a JSON array of Objects with the feed info as well as the reason why it was invalidated.
+feeds.get('/invalid', async (req, res, next) => {
+  let keys = [];
+  try {
+    keys = await getInvalidFeeds();
+  } catch (error) {
+    logger.error({ error }, 'Unable to get invalid feeds from Redis');
+    next(error);
+  }
+  const invalidsFeeds = keys.map(async (element) => {
+    const feedInfo = await Feed.byId(element.hashId);
```

Comment:
Do we need the feed details?  If you already have the `id` and `reason`, that's probably enough.  I would skip getting all the feeds by id.

Reply:
that is great, it will be cleaner